src/Nix/Builtins.hs

```diff
@@ -244,6 +246,16 @@ builtinsList = sequence
   , add  Normal   "readDir"          readDir_
   , add  Normal   "readFile"         readFile_
   , add2 Normal   "findFile"         findFile_
+  {-
+  , add  Normal   "fetchGit"         fetchGit
+  , add  Normal   "fetchMercurial"   fetchMercurial
+  , add  Normal   "filterSource"     filterSource
+  , add  Normal   "fromTOML"         fromTOML
+  -}
+  , add  Normal   "getContext"       getContext
+  {-
+  , add  Normal   "path"             path
+  -}
```

Comment:
Is this list supposed to be alphabetized?

the TODO elements and 'nixPath' are out of order.

Reply:
Oh that's funny. Something tampered with my internal Ord String instance :-)

Reply:
Turns out I was not completely bind. The order was not correct anyway. Fixed in https://github.com/haskell-nix/hnix/pull/757/commits/1a3a2abbc4e091d3b666578ed313488aa8e10641